### DISCUSSION TITLE
Error with the Storyblok blog example code

### DISCUSSION THREAD

**Discussion by rishibhargava** [vercel/next.js#37742] [Help]
When I try the standard example of Storyblok blog - I get the following error related to nextjs/image. 

Error: Failed to parse src "//a.storyblok.com/f/162244/6048x3402/205620be68/surface-h0bmuj6xcn4-unsplash.jpg" on `next/image`, protocol-relative URL (//) must be changed to an absolute URL (http:// or https://)

Any ideas on what might be going on? This is standard the example from here - https://github.com/vercel/next.js/tree/canary/examples/cms-storyblok

**Comment by icyJoseph** [comment]
Hi,

When you add an image to Storyblok, it looks like they have made it so that you can optionally prepend the protocol to the URL, and it is set to false by default, so the images URLS are protocol-relative, aka start with `//`, here's more info on what are those, https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#linking_to_relative_urls.  

### Fix in the Storyblok dashboard 

To fix this, in the Image uploading section, you'll find a `cog`, click on it and you should see this screen:

<img width="486" alt="Screenshot 2022-06-16 at 14 05 55" src="https://user-images.githubusercontent.com/21013447/174066071-1d399e92-efd5-4ac0-bcd0-97e33871b335.png">

<img width="465" alt="Screenshot 2022-06-16 at 14 02 11" src="https://user-images.githubusercontent.com/21013447/174065505-6aa911dc-8ca3-4f9a-9ee3-54d0a1c74449.png">

Make sure the `https` option is selected, I'd recommend re-uploading the image because it seems that it doesn't automatically upgrade to `https` if it was not selected, and then **republish** the Post once again.

### next.config.js

However the above uncovers yet another error, you'll need to create a `next.config.js` file with this content:

```js
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  images: {
    domains: ["a.storyblok.com"],
  },
};

module.exports = nextConfig;
```

And now it should work. Perhaps you want to open a PR with these updates?
